python/fastquant/strategies/base.py

```diff
@@ -30,14 +33,60 @@
     SELL_PROP,
     SHORT_MAX,
 )
+from fastquant.utils.strategy_logging import StrategyLogger
 
 
 class BaseStrategy(bt.Strategy):
     """
     Base Strategy template for all strategies to be added to fastquant
+
+    Attributes
+    ----------
+    init_cash : float
+        inital cash
+    buy_prop : float
+        buy proportion, percentage of available units to buy, setting to 1.0 buys all afforded units
+    sell_prop : float
+        sell proportion, percentage of owned units to sell
+    commission: float
+        Percentage of comission
+    stop_loss : float
+        stop loss percent
+    stop_trail : float
+        stop trail percent
+    execution_type : str {`market`,`close`} (default=`market`)
```

Comment:
@jbdelmundo Any specific reason why you decided to leave out the `open` option that was previously implemented? :smile:

Reply:
Actually I am having trouble making the backtest use the next opening price. 
I have tried using Market and Close orders in backtrader but it seems both use the closing price.
For Close orders, it uses the next day (or bar) close price, and for Market orders, it uses the current closing price

I am using the Ternary Strategy to manually provide buy and sell signals and review them on the order history